### PR TITLE
Make deployable

### PR DESCRIPTION
**Pull request by deepstqte**
@chriddyp In order to make this deployable we were missing:

- A config file
- A worker for celery beat in the Procfile
- and gunicorn pip package in the requirements file

I also made a v0.0.1 of a deployment helper, it's usable, but for from being complete, I made it on the go to help me deploy more often, faster... so please have a look at it, and feel free to suggest any improvements.

cc @cldougl

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 web: gunicorn app:server --log-file=-
 worker: celery -A tasks worker --loglevel=info
+worker: celery -A tasks beat --loglevel=info
```
Comment (cldougl) [inline Procfile]: does this work for deployment or does the beat need to be added as a tag in the worker above?

Comment (gzork) [reply Procfile]: This ended up working for some reason, also I saw that `--beat` tag is not recommended for a `production` environment.

Comment (cldougl) [reply Procfile]: nice!! 🎉

**Comment by cldougl** [comment]
lgtm 💃

**Pull request merged**